### DISCUSSION TITLE
`withCount` is lost on model after component re-render

### DISCUSSION THREAD

**Discussion by Spriithy** [livewire/livewire#4505] [1. Help] [answered]
Hi !

I'm using Livewire in one of my projects at work and stumbled across a weird edge case (I believe ?) where updating a property using a `wire:click` function call removes the `withCount` property of one of the models loaded in the component.

```php
// StoreCollectivity.php
...
    public $searchCollectivities = [];
    public $collectivities = [];

    public function mount()
    {
        $this->searchCollectivities = Collectivity::select(['id', 'name'])->withCount('mandates')->get();
    }
...

    public function deselectAll()
    {
        $this->collectivities = [];
    }
...
```

Upon calling the `deselectAll` method from the view, the `$this->searchCollectivities` models no longer have the `mandates_count` property and all other fields besides `id` and `name` are loaded.

Do you guys have any idea how to circumvent this ?

Thanks for your help !

**Comment by Spriithy** [comment, marked as answer]
My bad, I searched the issue list, not the discussions... found out what I wanted in https://github.com/livewire/livewire/discussions/2950 

Too bad these pages aren't indexed by google somehow ?

**Reply by joshhanley** [reply]
@Spriithy glad you found that other discussion. Yeah seems strange that google doesn't index them!
